spec/rspec/matchers/composable_spec.rb

```diff
@@ -12,6 +12,21 @@ module Matchers
         }.to fail_with(STDOUT.inspect)
       end
 
+      it "does not blow up when surfacing descriptions from an unreadable Range object" do
+        infinity ||= (1.0/0.0)
```

Comment:
in what circumstance would `infinity` already be set?  Isn't this equivalent to `infinity = (1.0/0.0)`?  Why use `||=`?